monkey/tests/unit_tests/infection_monkey/credential_collectors/test_mimikatz_collector.py

```diff
@@ -110,3 +117,26 @@ def test_pypykatz_result_parsing_no_secrets(monkeypatch):
     collected_credentials = collect_credentials()
     assert len(collected_credentials) == 1
     assert collected_credentials == expected_credentials
+
+
+def test_mimikatz_credentials_stolen_event_published(monkeypatch):
+    mock_event_queue = MagicMock(spec=IEventQueue)
+    mock_subscriber = mock_pypykatz_handler = Mock()
+    monkeypatch.setattr(
+        "infection_monkey.credential_collectors.mimikatz_collector.pypykatz_handler",
+        mock_pypykatz_handler,
+    )
+
+    for event_tag in MIMIKATZ_EVENT_TAGS:
+        mock_event_queue.subscribe_tag(event_tag, mock_subscriber)
```

Comment:
You don't need to subscribe, you just need to validate that the correct data got published.